npm-shrinkwrap.json

```diff
@@ -5042,8 +5042,8 @@
 			}
 		},
 		"color-studio": {
-			"version": "github:automattic/color-studio#302bab1a6279ce539cd47cf369bf307a0d300d71",
```

Comment:
this was the important part for CI :)

Reply:
The next `npm run update-deps` would hurt 🙂

Reply:
indeed!